soft_bfddpd/bfddpd.c

```diff
@@ -71,7 +64,8 @@ static void bfddp_handle_message(struct events_ctx *ec, struct bfddp_ctx *bctx);
 /**
  * Handle common read/write events.
  */
-static int bfddp_event(struct events_ctx *ec, int fd, short revents, void *arg);
```

Comment:
would it be possible to move the impl. of these functions to here, to avoid unnecessary fwd decl.?

Reply:
I'd agree with this, but the code is already there and it would generate bigger changes (and possible conflicts) :(